# Copyright (c) 2023 Real-Time Development Center (RTDC) Project
# All rights reserved.

float32 input_voltage         # volts
float32 transmit_voltage # volts
float32 transmit_current # amps